platform/core/src/services/HangingProtocolService/HPMatcher.js

```diff
@@ -74,14 +74,6 @@ const match = (
       errorMessages = ['Something went wrong during validation.', e];
     }
 
-    console.log(
```

Comment:
Could you move this into log4js or the like instead of removing - comment it out for now, and mark it as a change for log4js rather than removing entirely becuase it is really hard to fix hanging protocols without logs indicating why things didn't work or matched unexpectedly.